dynamic-config.md

```diff
@@ -207,6 +207,7 @@ The following TiKV configuration items can be modified online:
 | `backup.num-threads` | The number of backup threads (supported since v4.0.3) |
 | `split.qps-threshold` | The threshold to execute `load-base-split` on a Region. If the QPS of read requests for a Region exceeds `qps-threshold` for a consecutive period of time, this Region should be split.|
 | `split.byte-threshold` | The threshold to execute `load-base-split` on a Region. If the traffic of read requests for a Region exceeds the `byte-threshold` for a consecutive period of time, this Region should be split. |
+| `split.region-cpu-overload-threshold-ratio` | The threshold to execute `load-base-split` on a Region. If the CPU usage in the Unified Read Pool for a Region exceeds the `region-cpu-overload-threshold-ratio` for a consecutive period of time, this Region should be split. (supported since v6.2.0) |
```

Comment:
Does "for a consecutive period of time" mean 10 seconds?

Reply:
> Does "for a consecutive period of time" mean 10 seconds?

Yes, fixed in https://github.com/pingcap/docs/pull/9656/commits/46fa37c9e65ee3a83b77e29fa7c84dbc496c6c7d